### ISSUE TITLE
Is it abandoned? Any active fork?

### ISSUE THREAD

**Issue by vitaly-zdanevich** [cerebroapp/cerebro#692]
Last commit 8 month ago.

**Comment by aalonsolopez** [comment]
Seems abandoned yep... Maybe someone with new and fresh ideas can fork this.

**Comment by aalonsolopez** [comment]
Furthermore, it's original creator is working for Raycast which aims to be pretty similar to the initial idea of this

**Comment by vitaly-zdanevich** [comment]
Raycast is macOS only :(

**Comment by aalonsolopez** [comment]
Yeah. I'm using cerebro and seems to work well, but it's sad that looks discontinued.

**Comment by PeterDaveHello** [comment]
Hi @lubien @ogustavo-pereira,

Is there anything that we can do to help continue the development of this awesome project?

**Comment by ogustavo-pereira** [comment]
I didn't have time to dedicate to this project this year. I'll see next year how it goes, but any help is welcome.

**Comment by aalonsolopez** [comment]
But is any prior task to do in which we can help you?

**Comment by ogustavo-pereira** [comment]
I'm using the help-wanted label

Anyone with this label can attack and create a pull request
